library/oneview_enclosure_group_facts.py

```diff
@@ -125,8 +125,6 @@ def execute_module(self):
         if name:
             if self.current_resource:
                 enclosure_groups = self.current_resource.data
-                if "configuration_script" in self.options:
-                    facts["enclosure_group_script"] = self.current_resource.get_script()
         else:
```

Comment:
Is this valid only for c7000 hardware..?

Reply:
no, it applies for synergy also